onnx/defs/schema.cc

```diff
@@ -502,7 +502,22 @@ OpSchema& OpSchema::AllowUncheckedAttributes() {
   allows_unchecked_attributes_ = true;
   return *this;
 }
-
+OpSchema& OpSchema::AddOpAnnotation(std::shared_ptr<OpAnnotation> annotation) {
```

Comment:
Do we really need this?

Reply:
I think we should keep this. What I was envisioning is that if you have a some family of operators that contain a lot of annotations. You would create that a new OpAnnotation class and pass that instance of that class in.

Reply:
Also in the future I was thinking OpAnnotation could itself contain attributes. For example if the OpAnnotationFlag is Bounded, OpAnnotation would have a min/max attribute.